MVC-DataExploration/Controllers/ProductController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace MVC_DataExploration.Controllers
+{
+    public class ProductController : Controller
+    {
+        // GET: ProductController
+        public ActionResult Index(Models.Product mp)
+        {
+            return View();
```

Comment:
The View still needs Product data, so pass the view a copy of the Products dataset from the database context. You will need to first instantiate a reference to the database context, then use it to pass the data to the view like so...

return View(dbc.Products);